orchestrator/build.gradle

```diff
@@ -15,11 +15,11 @@
  */
 
 plugins {
-    id 'java'
-    id 'de.gliderpilot.semantic-release' version '1.4.0'
+    id "java"
+    id "de.gliderpilot.semantic-release" version "1.4.0"
```

Comment:
Can we extract this version too for consistency?

Reply:
Missed that one. Thanks!

Reply:
Actually lets talk about this one. It seems that specifically for plugins you cannot use some parts of groovy

see https://docs.gradle.org/4.10.3/userguide/plugins.html#sec:plugins_block